internal/models/running_output.go

```diff
@@ -138,7 +138,7 @@ func (ro *RunningOutput) Write() error {
 }
 
 func (ro *RunningOutput) write(metrics []telegraf.Metric) error {
-	if len(metrics) == 0 {
+	if metrics == nil || len(metrics) == 0 {
```

Comment:
This surprises me. I think `len` is [safe to call on nil slices](https://golang.org/ref/spec#Length_and_capacity).